Vector.cpp

```diff
@@ -1,8 +1,21 @@
 #include "Vector.h"
 #include <assert.h>
+#include <iostream>
+
+using namespace std;
 
 namespace Micromouse
 {
+	direction operator%(const direction &dir1, const direction &dir2)
```

Comment:
I don't think this works as you intended
doing it by hand I get
`NE % S = SW`
`W % W = S`
`NW % N = NW`